src/logic.js

```diff
@@ -265,6 +264,21 @@ define([
             });
             return messages;
         },
+        addReferences: function (mug, property, value) {
+            // get absolute paths from mug property's value
+            var isLabel = false,
```

Comment:
Better idea: add a function to the `ItextItem` type: `ItextItem.mapExpressions(func)`, which would encapsulate `$('<div>').append(mug.p[property].get()).find('output').map(func)`. This would be cleaner since `ItextItem` knows about its implementation details like multiple languages, output tags, etc. and those things need to be hard-coded into the logic manager. This design would allow any mug property to use this protocol to embed multiple expressions in an opaque property value.

If you do that, this line becomes
```js
var isLabel = (!value && mug.p[property] && _.isFunction(mug.p[property].mapExpressions)),
```